lookout/style/typos/config.py

```diff
@@ -0,0 +1,62 @@
+import multiprocessing
+import pathlib
+
+
+DEFAULT_DATA_DIR = pathlib.Path(__file__).parent / "data"
+DEFAULT_CONFIG = {
```

Comment:
Let's call it `DEFAULT_PRETRAIN_CONFIG` or something to not confuse with `DEFAULT_CONFIG` which is for Analyzer in FormatAnalyzer